_data/communication.yml

```diff
@@ -180,17 +186,20 @@ websites:
     - name: ToutApp
       url: https://www1.toutapp.com/
       twitter: toutapp
+      facebook: toutapp
       img: toutapp.png
       tfa: No
 
     - name: WhatsApp
       url: https://www.whatsapp.com/
       twitter: whatsapp
+      facebook: WhatsApp
```

Comment:
weird. seems to be working for me. have to be logged in, but that's the same requirement for messaging on `twitter`, so that shouldn't matter
![screenshot 2016-10-16 at 21 32 25](https://cloud.githubusercontent.com/assets/1159138/19425507/164005e0-93e8-11e6-9d62-512ba08e4780.png)